website/src/panel-layout.tsx

```diff
@@ -93,6 +130,13 @@ export const PanelLayout = (p: {
 export const PanelContent = (p: {
   panel: PanelType
   word: Dailp.FormFieldsFragment
+  groupedOptions: {
```

Comment:
Maybe pull this into its own type definition in case we need to use it elsewhere? Its a bit nested for an inline type definition.